### PR TITLE
typofix - https://github.com/vlajos/misspell_fixer

### PR DESCRIPTION
**Pull request by vlajos**

### CODE REVIEW

**Comment by rnicoll** [comment]
utACK, looks simple and a good bit of cleanup

File: doc/Doxyfile [hunk 1]
```diff
@@ -870,7 +870,7 @@ HTML_FILE_EXTENSION    = .html
 # standard header. Note that when using a custom header you are responsible 
 # for the proper inclusion of any scripts and style sheets that doxygen 
 # needs, which is dependent on the configuration options used. 
-# It is adviced to generate a default header using "doxygen -w html 
+# It is advised to generate a default header using "doxygen -w html
```
Comment (fanquake) [inline doc/Doxyfile]: Don't change this, it will be fixed if/when we update the Doxyfile.

File: doc/release-notes/release-notes-0.10.0.md [hunk 2]
```diff
@@ -370,7 +370,7 @@ RPC:
 - `b5ec5fe` update getnetworkinfo help with subversion
 - `ad6e601` RPC additions after headers-first
 - `33dfbf5` rpc: Fix leveldb iterator leak, and flush before `gettxoutsetinfo`
-- `2aa6329` Enable customising node policy for datacarrier data size with a -datacarriersize option
```
Comment (luke-jr) [inline doc/release-notes/release-notes-0.10.0.md]: This is an Americanisation (also, release notes are historical/immutable)

**Comment by vlajos** [comment]
Done.. I have just reverted doc/release-notes and doc/Doxyfile.

File: src/qt/bitcoinstrings.cpp [hunk 3]
```diff
@@ -225,7 +225,7 @@ QT_TRANSLATE_NOOP("bitcoin-core", "Error loading wallet.dat: Wallet requires new
 QT_TRANSLATE_NOOP("bitcoin-core", "Error opening block database"),
 QT_TRANSLATE_NOOP("bitcoin-core", "Error reading from database, shutting down."),
 QT_TRANSLATE_NOOP("bitcoin-core", "Error"),
-QT_TRANSLATE_NOOP("bitcoin-core", "Error: A fatal internal error occured, see debug.log for details"),
+QT_TRANSLATE_NOOP("bitcoin-core", "Error: A fatal internal error occurred, see debug.log for details"),
```
Comment (Diapolo) [inline src/qt/bitcoinstrings.cpp]: This won't work as bitcoinstrings.cpp is generated during build. You need to fix the spelling just in the "normal" source.

File: src/rpcserver.cpp [hunk 4]
```diff
@@ -606,7 +606,7 @@ void StartRPCThreads()
         LogPrintf("No rpcpassword set - using random cookie authentication\n");
         if (!GenerateAuthCookie(&strRPCUserColonPass)) {
             uiInterface.ThreadSafeMessageBox(
-                _("Error: A fatal internal error occured, see debug.log for details"), // Same message as AbortNode
+                _("Error: A fatal internal error occurred, see debug.log for details"), // Same message as AbortNode
```
Comment (Diapolo) [inline src/rpcserver.cpp]: This one is it ;).

File: src/leveldb/port/port_posix.h [hunk 5]
```diff
@@ -71,7 +71,7 @@
 
 #if defined(OS_ANDROID) && __ANDROID_API__ < 9
 // fdatasync() was only introduced in API level 9 on Android. Use fsync()
-// when targetting older platforms.
```
Comment (laanwj) [inline src/leveldb/port/port_posix.h]: Please don't change the files inside `src/leveldb` - they come verbatim from upstream, merging this change would make it harder to keep the subtree up to date

**Comment by laanwj** [comment]
Looks good to me, apart form leveldb nit

File: src/leveldb/port/port_posix.h (hunk 5 above)
Comment (vlajos) [inline src/leveldb/port/port_posix.h]: OK. I have removed these 2 changes.

**Comment by paveljanik** [comment]
Also please remove the URL from the commit message.

**Comment by vlajos** [comment]
I have removed the URL.
(But kept the tool's name. Let me know if this a problem.)

**Comment by btcdrak** [comment]
ACK

**Comment by jonasschnelli** [comment]
ACK

**Comment by dcousens** [comment]
ACK

**Pull request merged**